jenkins-build-driver/src/main/java/org/jboss/pnc/jenkinsbuilddriver/BuildJob.java

```diff
@@ -35,19 +44,19 @@ public boolean configure(RunningEnvironment runningEnvironment, boolean override
         try {
             job = jenkinsServer.getJob(jobName);
         } catch (IOException e) {
-            throw new BuildDriverException("Cannot check for existing job.", e);
+            log.info("No job exists, so there will be created new one.");
```

Comment:
Each exception you catch must be handled, in case of IOException, I see two options, you can retry or fail.
I think in our case it is safe to fail and cancel the build by trowing en exception. The error will be logged in BuildRecord which is better than system log that won't be available to users.